data/ms/legislature/Johnathan-Ray-Lancaster-d0a3f800-01b9-4442-94e2-676c0a364450.yml

```diff
@@ -1,25 +1,34 @@
 id: ocd-person/d0a3f800-01b9-4442-94e2-676c0a364450
 name: Johnathan Ray Lancaster
+family_name: Lancaster
 email: jlancaster@house.ms.gov
 image: http://billstatus.ls.state.ms.us/members/house/lancaster.jpg
 party:
-- name: Democratic
+- end_date: '2022-12-29'
```

Comment:
@NewAgeAirbender This person has an end date of January 2, 2024. In 2021 they switched from being a democrat to republican. So their party should be republican, although it looks like both will be listed here.